### PR TITLE
Update websocket.ksy

### PR DESCRIPTION
**Pull request by bspeice**
Handle initial-frame issues as in #143.

### CODE REVIEW

File: network/websocket.ksy [hunk 1]
```diff
@@ -10,12 +10,54 @@ doc: |
...
+  - id: trailing_frames
     type: dataframe
+    if: initial_frame.finished != true
     repeat: until
     repeat-until: _.finished
 
 types:
+  initial_frame:
+    seq:
+      - id: finished
```
Comment (KOLANICH) [inline network/websocket.ksy]: I see lot of duplication.

**Comment by bspeice** [comment]
Should look better now. I'm still parsing as initial/trailing frames, but the header has been factored out.

File: network/websocket.ksy [hunk 2]
```diff
@@ -74,3 +94,4 @@ enums:
     0xD: reserved_control_d
     0xE: reserved_control_e
     0xF: reserved_control_f
+
```
Comment (GreyCat) [inline network/websocket.ksy]: Please be more careful about whitespace ;)

**Comment by GreyCat** [comment]
LGTM! Thanks!

**Pull request merged**